shell.lint.cloudbuild.yaml

```diff
@@ -5,8 +5,8 @@ steps:
   args:
   - -c
   - | # Recursively find shell scripts (not in testing dir)
-    find . -name '*.sh' -not -path './testing/*' \
+    find . -name "*.sh" -not -path "./testing/*" \
```

Comment:
globs strike again?

Reply:
This seems safer and more consistent. Can we chat about this in our next 1:1?

Reply:
In practice it doesn't seem to have an effect, but for overall consistency should I remove the quotes?